web_app/dist/main.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <link rel="icon" type="images/png" href="images/icon.png" />       
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="main.css">
+    </head>
+
+    <body>
+
+        
+    <div class="navbar">
+        <img id="logo" src="images/iconSmall.png" alt="TaskViewer Logo">
+        <a href="#Home">Home</a>
```

Comment:
`href`s are probably unnecessary, I'm thinking we'll probably control this using full JS, we'll see if we want to change the URL at some later time